### ISSUE TITLE
`freezeOptions()` can create issues in some browsers

### ISSUE THREAD

**Issue by alexturpin** [autoNumeric/autoNumeric#622]
We've noticed in our app that some browsers are throwing a `TypeError: Object.freeze called on non-object` on [this line in `freezeOptions` in `AutoNumericOptions.js`](https://github.com/autoNumeric/autoNumeric/blob/next/src/AutoNumericOptions.js#L871):

```js
if (!AutoNumericHelper.isIE11() || options.valuesToStrings[valuesToStringObjectName] !== null) {
    Object.freeze(options.valuesToStrings[valuesToStringObjectName]);
}
```

Further investigation revealed that some browsers throw an error when the parameter of `Object.freeze` is `null` or not an object. `valuesToStrings` by default contains a property `"none"` that is `null`.

Upon quick observation it would seem that this should be guarded by the above condition, but if you look closely, it's interpreted as "if we aren't on IE11 or the value isn't null, freeze it". I believe it should instead read like this:

```js
if (!AutoNumericHelper.isIE11() && options.valuesToStrings[valuesToStringObjectName] !== null) {
    Object.freeze(options.valuesToStrings[valuesToStringObjectName]);
}
```

I'll submit a pull request with the change.

We are on 4.5.0 but this issue should be present in later versions as well.

**Comment by AlexandreBonneau** [comment]
Hey Alex, thanks for reporting this issue, and the solution that goes with it :) :+1:

**Comment by AlexandreBonneau** [comment]
You can access your fix in `v4.5.3` @alexturpin

**Comment by alexturpin** [comment]
Thanks for the prompt merge and release @AlexandreBonneau !
